src/pages/tBTC/Bridge/DepositDetails.tsx

```diff
@@ -180,129 +179,111 @@ export const DepositDetails: PageComponent = () => {
-                        : "Minting"}
-                    </Box>
-                    {mintingProgressStep !== "completed" && ` - In progress...`}
-                  </BodyLg>{" "}
-                  <InlineTokenBalance
-                    tokenAmount={amount || "0"}
-                    tokenSymbol="tBTC"
-                    withSymbol
-                    ml="auto"
-                  />
-                </Flex>
-                {mintingProgressStep !== "completed" && (
-                  <Alert status="info" my={6}>
-                    <AlertIcon />
-                    <AlertDescription>
-                      It is safe to close this window. Minting will continue as
-                      a background process and will not be interrupted.
-                    </AlertDescription>
-                  </Alert>
-                )}
-                <StepSwitcher />
+      {(isFetching || !data) && !error && <BridgeProcessDetailsPageSkeleton />}
+      {error && <>{error}</>}
+      {!isFetching && !!data && !error && (
+        <>
+          <Stack
+            direction={{
+              base: "column",
+              xl: "row",
+            }}
+            divider={<StackDivider />}
+            spacing={4}
+          >
+            <Flex flexDirection="column" w={{ base: "100%", xl: "65%" }}>
+              <Flex mb="4" alignItems="center" textStyle="bodyLg">
+                <BodyLg>
+                  <Box
+                    as="span"
+                    fontWeight="600"
+                    color={depositStatusTextColor}
+                  ></Box>
+                </BodyLg>{" "}
+                <InlineTokenBalance
+                  tokenAmount={amount || "0"}
+                  tokenSymbol="tBTC"
+                  withSymbol
+                  ml="auto"
+                />
               </Flex>
-              <Flex
-                w={{ base: "100%", xl: "35%" }}
-                mb={{ base: "20", xl: "unset" }}
-                direction="column"
+              {mintingProgressStep !== "completed" && (
+                <Alert status="info" my={6}>
+                  <AlertIcon />
+                  <AlertDescription>
+                    It is safe to close this window. Minting will continue as a
+                    background process and will not be interrupted.
+                  </AlertDescription>
+                </Alert>
+              )}
+              <StepSwitcher />
+            </Flex>
+            <Flex
+              w={{ base: "100%", xl: "35%" }}
+              mb={{ base: "20", xl: "unset" }}
+              direction="column"
+            >
+              <LabelSm mb="8" mt={{ xl: 2 }}>
+                Transaction History
+              </LabelSm>
+              <Badge
+                size="sm"
+                colorScheme="yellow"
+                variant="solid"
+                display="flex"
+                alignItems="center"
+                alignSelf="flex-start"
+                mb="4"
```

Comment:
Hmm, look like we have two right sections right now:

<img width="1098" alt="image" src="https://github.com/threshold-network/bitcoin-on-base/assets/40306834/25370f1c-eb53-4f40-b1bb-08bbf6caf9da">

I think this issue might be related to this comment: https://github.com/threshold-network/bitcoin-on-base/pull/18#discussion_r1508449344. We could combine the fix for both by setting an `undefined` for right section (probably only for depoist details page) and render the correct one inside the actual component likt it is done right now, in this PR.

Reply:
In this PR I focused only on Success step itself. I didn't touch anything around (like removing existing UI parts) because it's a part of Deposit Details view and it's already done in #18, good example why I'm missing Storybook here 😆 . 
About your suggestion - I elaborated on it in reply to the comment you mention.